modules/AlphaWalletCore/AlphaWalletCore/WebImageURL.swift

```diff
@@ -79,7 +79,7 @@ public enum WebImageURL: Codable, Hashable, Equatable, CustomStringConvertible {
     }
 
     public init?(string: String, withUrlRewriting: Bool = true, rewriteGoogleContentSizeUrl size: GoogleContentSize = .s750) {
-        guard let url = URL(string: string) else { return nil }
+        guard let url = URL(string: string.replacingOccurrences(of: " ", with: "%20").trimmed) else { return nil }
```

Comment:
Maybe trim first then replace is better? — faster and I don't know what happens if there are leading or trailing space character that are converted to "%20"